### ISSUE TITLE
Ruby LSP never tries to activate

### ISSUE THREAD

**Issue by bringel** [Shopify/vscode-ruby-lsp#502]
Nothing happens after installing the extension and switching to a ruby file in my project. I was expecting to see the popup showing progress installing gems and such but it never appeared. Similarly if I look at the runtime stats tab for the extension, it says it never activated

<img width="2311" alt="image" src="https://user-images.githubusercontent.com/305085/228593028-b4c33c83-cac3-4e5d-b029-e72c1b667b6c.png">

This is on a fresh vscode profile with no other extensions installed

**Comment by vinistock** [comment]
The extension gets activated if there's a `Gemfile.lock` in the repo or if you open a Ruby file in the UI. Did you do any of those two?

If so, is there anything in the output tab under the Ruby LSP channel?

If you open a Ruby file and open the language status centre (the curly braces next to the language mode `Ruby`), does it display any errors?

**Comment by bringel** [comment]
Yeah we have a `Gemfile.lock` and I tried opening various ruby files. I don't even have the curly braces next to `Ruby`. For reference we're using `chruby` and this folder also has a `.ruby-version` file in it

**Comment by bringel** [comment]
I am not 100% sure what changed (so I know this won't be particularly helpful if someone finds this issue), but this is working on this repository now. I did try to open a totally blank project when debugging, and although that also didn't seem to work, it may have triggered the gem to start downloading properly
